core/src/main/scala/org/apache/spark/TestUtils.scala

```diff
@@ -236,7 +236,11 @@ private[spark] object TestUtils {
    * Test if a command is available.
    */
   def testCommandAvailable(command: String): Boolean = {
-    val attempt = Try(Process(command).run(ProcessLogger(_ => ())).exitValue())
+    val attempt = if (Utils.isWindows) {
+      Try(Process(command).run(ProcessLogger(_ => ())).exitValue())
```

Comment:
@dongjoon-hyun, I manually tested. `command` -> `$"WHERE $command"` seems working fine in Windows with Scala 2.13.